ursula_cli/shell.py

```diff
@@ -195,7 +195,8 @@ def _run_heat(args, hot):
     CREDS = {
         'username': os.environ['OS_USERNAME'],
         'password': os.environ['OS_PASSWORD'],
-        'tenant_name': os.environ['OS_TENANT_NAME'],
+        'tenant_name': (os.environ.get('OS_TENANT_NAME') or
+                        os.environ['OS_PROJECT_NAME']),
```

Comment:
We tried that first, but if OS_PROJECT_NAME doesn't exist you get an error here, because python apparently tries to resolve both sides of this in the get call. So in the case where OS_TENANT_NAME exists but OS_PROJECT_NAME doesn't, you get a traceback.